spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -218,6 +225,13 @@ def assert_specific_vm_powered_off
     )
 
     expect(v.ext_management_system).to eq(@ems)
+    expect(v.orchestration_stack).to have_attributes(
+       :type        => ManageIQ::Providers::Vmware::CloudManager::OrchestrationStack.name,
+       :ems_ref     => "vapp-bfd40328-a383-4b0e-9130-66edd590f4ab",
+       :name        => "vApp_admin_3",
+       :description => "vApp_admin_3",
+       :status      => "unknown",
+    )
```

Comment:
And for this if you're just checking that the vapp is set you can `expect(v.orchestration_stack.ems_ref)`, testing that the vapp attributes are saved correctly should be done in its own test.